src/main/resources/application.yml

```diff
@@ -11,6 +11,9 @@ spring:
   application:
     name: prison-api
 
+  cache:
```

Comment:
This config isn't actually needed.  Introducing the spring-boot-starter-cache and cache-api (both needed) automatically pulls in jcache.  However, I can imagine it would be easy to just remove the two jars and everything would "appear" to work ok, so I have added it in (leaving it in and removing the two jars will stop the application from starting, so is more of a safety net)

Reply:
CORRECTION - the cache-api is not required